adding_species_manuscript.md

```diff
@@ -389,31 +391,238 @@ Table of species/demographic scenarios added/worked on by community since origin
+
+Our workflow starts with considerations of git and version control. Our first step will be to create an upstream link to the version of the repo owned by popsim-consortium, then we will create a new branch of stdpopsim using git which will effectively keep track of the new features we will be adding
+
+```
+git remote add upstream https://github.com/popsim-consortium/stdpopsim.git
+git fetch upstream
+git checkout upstream/main
+git checkout -b mosquito
+```
+
+now any changes that I commit, will only be committed to the branch called ```mosquito```.
+
+stdpopsim has a few utilities that play well with Ensembl, including one that we will use to create some code templates for us to buildout a new species. This utility lives on the "maintenance" side of stdpopsim which houses utilities that are not user facing.
+
+To add our species we will hand the maintenance command line interface an Ensembl species ID and it will go and pull down some essential information about the genome:
+
+```
+python -m maintenance add-species Anopheles_gambiae
+```
+
+A partial list of the genomes housed on Ensembl [can be found here](https://metazoa.ensembl.org/species.html).
+
+The next steps are going to be to flesh out the code templates that the maintenance code just wrote for us in our branch. 
+
+The function call above has created a few new files in the directory structure of stdpopsim, in particular at the level of ```stdpopsim/catalog``` this utility has created a new directory called AnoGam, which contains three files. The 3 + 3 naming scheme for this directory (and all following references in stdpopsim) is automatically parsed by the maintenance script:
+
+```
+├── catalog
+│   ├── AnoGam
+│   │   ├── __init__.py
+│   │   ├── genome_data.py
+│   │   └── species.py
+```
+* The ```__init__.py``` contains information about XXXXXXXXXX 
+* The ```genome_data.py``` contains a data dictionary which has slots for the assembly accession number, the assembly name, and a dict representing the chromosome names and their associated lengths.
+* The ```species.py``` is the file that we will need to edit with our species specific information. The basics that we will need here include:
+  * 1) *chromosome-specific* recombination rates (could also be single rate)
+  * 2) a genome-wide average *mutation rate*
+  * 3) a *generation time* estimate (in years)  
+  * 4) a default *population size*
+  * 5) literature citations that one can point to for the above and the assembly
+  
+- Test the code while building the files
+
+```stdpopsim``` in part guarantees code quality through the use of unit testing. Basic sanity tests for this new species will be completed through QC over in the ```tests/test_AnoGam.py``` file created by the command ```python -m maintenance add-species Anopheles_gambiae``` above.
+
+To run the tests in stdpopsim uses the pytest module. We will quickly run the tests that our maintenance function has stubbed out and expect it to error out:
```

Comment:
"We will quickly run the tests that our maintenance function has set up and expect it to give us an error because we have not yet filled out ```species.py``` with the necessary species information." ("stub out" and "error out" are a little jargon-y)

Reply:
done